### DISCUSSION TITLE
Object position

### DISCUSSION THREAD

**Discussion by dudckd502** [nesrak1/UABEA#308] [Q&A]
Hello.

I am using UABEA tool well, thank you for making a good tool.

I wanted to adjust the position of the object in the game, so I looked for it with a tool, but I couldn't fix it separately.

![2024-02-21 22 31 31](https://github.com/nesrak1/UABEA/assets/56987222/7b55df4c-9a80-418c-a047-fc1a2161589c)

Does UABEA not support the ability to modify with this tool yet?

**Comment by nesrak1** [comment]
There are multiple options. If you right click and choose edit data on any field in that component without an equal sign, you can directly edit the asset in text form.
If you choose the visit asset option instead, it will select the asset in the list view where you can export/import dump.

**Reply by dudckd502** [reply]
Aha! I guess I was trying the wrong thing. Thank you for your quick response! :)
